src/app/material-docs-app.scss

```diff
@@ -16,3 +16,11 @@ app-navbar {
   position: relative;
   z-index: 10;
 }
+
+.pull-right {
+  float: right;
+}
+
+.pull-left {
+  float: left;
+}
```

Comment:
Rather than using float, these should use flex stylings to put things on the left/right.